libs/sdk-backend-tiger/src/backend/workspaces/index.ts

```diff
@@ -66,16 +72,14 @@ class TigerWorkspaceQuery implements IWorkspacesQuery {
     private searchWorkspaceDescriptors =
         (search?: string) =>
         (results: IWorkspaceDescriptor[]): IWorkspaceDescriptor[] => {
```

Comment:
We probably don't have to change this function, as filtering by `parentWorkspaceId` should be done entirely in the BE.

Reply:
Removing this change. Thanks!